common/src/main/scala/azure/apns/Body.scala

```diff
@@ -0,0 +1,23 @@
+package azure.apns
+
+import play.api.libs.json._
+
+import scala.collection.breakOut
+
+case class Body(aps: APS, customProperties: Map[String, String])
+object Body {
+  implicit val jf = new Format[Body] {
+    override def writes(o: Body): JsValue =
+      Json.toJson(o.customProperties.mapValues(JsString(_)) + ("aps" -> Json.toJson(o.aps)))
+
+    override def reads(json: JsValue): JsResult[Body] = for {
```

Comment:
If you're writing your own `reads` function, why passing by that extra step of deserializing through `BodyWithoutCustom`?

I think something in the effect of `(json / "aps").as[APS]` may do